linux_os/guide/system/accounts/accounts-session/accounts_tmout/rule.yml

```diff
@@ -1,14 +1,21 @@
 documentation_complete: true
 
-prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4,sle15,wrlinux1019,rhcos4
+prodtype: fedora,ol7,ol8,rhel7,rhel8,rhv4,sle12,sle15,wrlinux1019,rhcos4
 
 title: 'Set Interactive Session Timeout'
 
 description: |-
     Setting the <tt>TMOUT</tt> option in <tt>/etc/profile</tt> ensures that
     all user sessions will terminate based on inactivity. The <tt>TMOUT</tt>
+    {{% if product == "sle12" %}}
+    setting in <tt>/etc/profile.d/autologout.sh</tt> should read as follows:
```

Comment:
I understand that SLE12 STIG illustrates the configuration using the file `etc/profile.d/autologout.sh`. But I don't see really the benefit of adding this extra code in the rule description and the OVAL. Changes that are related to checking for `readonly TMOUT` and `export TMOUT` are debatable though. The purpose of the rule is to configure the `TMOUT` and it will check for both `/etc/profile` and `/etc/profile.d/*.sh` files which should cover the case specified by the STIG.

I'm in favor of keeping the rule more generic since we usually are not blindly following what STIG says, although we try to get the idea behind it right. So far, these `readonly TMOUT` and `export TMOUT` options have been absent from the content, RHEL7 STIG also contains the same configuration item and doesn't make use of these extra statements that prevent modification and make sure it's exported so it's taken into consideration by every session. If that's the case, the content can be updated for every product that this rule is part of.

Reply:
@ggbecker thank you, we were trying to follow the letter of the stig and then making sure that the description match what was in the resulting oval, but since RH does the same thing and does not call out the TMOUT items there is not pressing need for us to do it with SLES. And I agree that the less with under distro specific conditionals, the easier thing are to maintain. I will get that addressed on the next update.